spec/requests/api/admin/create_spec.rb

```diff
@@ -0,0 +1,100 @@
+RSpec.describe 'POST /api/admin/articles', type: :request do
+
+  describe 'Successfully with valid params and user' do
+    let(:journalist) { create(:user, role: 'journalist')}
+    let(:journalist_credentials) { journalist.create_new_auth_token }
+    let!(:journalist_headers) { { HTTP_ACCEPT: 'application/json' }.merge!(journalist_credentials) }
+
+    before do
+      post "/api/admin/articles",
+      params: {
+        article: {
+          title: "Article 1",
+          body: "Some content"
+        }
+      },
+      headers: journalist_headers
+    end
+
+    it 'returns a 200 response status' do
+      expect(response).to have_http_status 200
+
+    end
+  end
+
+  describe 'unsuccessfully with' do
+    describe 'no title and content' do
+    let(:journalist) { create(:user, role: 'journalist')}
```

Comment:
You are repeating code here. Place this (and others related to this one) in the main `describe` block

Reply:
ok

Reply:
you already have a journalist created, this line is not needed